src/component/CreateTeam.jsx

```diff
@@ -1,38 +1,40 @@
-const CreateTeam = ({ setTeamsDatabase, setTeamCode }) => {
+import '../index.css';
+// use this format if fetching
+//import { apiPath } from '../api';
+// fetch(apiPath('/teams'))
+const CreateTeam = ({ setTeamsDatabase }) => {
   const handleCreateTeam = async () => {
     try {
       const randomCode = Math.random().toString(36).substring(2, 8).toUpperCase();
 
-      const title = prompt("Enter new team name:");
+      const newName = prompt("Enter new team name:");
```

Comment:
is there reason for renaming title to newName?

Reply:
We had used title in the database as column name for what was actaully a team name. To more accurately describe what it is, we change title to newName.